app/src/main/java/com/alphawallet/app/ui/DappBrowserFragment.java

```diff
@@ -164,6 +166,7 @@
     private ImageView back;
     private ImageView next;
     private ImageView clear;
+    private ImageView refresh;
```

Comment:
Since this is the reload image view, recommend naming, `reload`, for consistency.

Reply:
Or are refresh and reload different things here?

Reply:
Not sure let's follow brave/chrome - which use 'reload' so you're right.